Monstarlab.EntityFramework.Extension/Repositories/BaseEntityRepository.cs

```diff
@@ -68,7 +68,7 @@ public virtual async Task<bool> DeleteAsync(TEntity entity)
 
         Context.Set<TEntity>().Remove(entity);
 
-        await Context.SaveChangesAsync();
+        //await Context.SaveChangesAsync();
 
         return true;
```

Comment:
Make the method not `async` and return:
```suggestion
        return Task.FromResult(true);
```